app/bundles/CampaignBundle/Form/Type/CampaignLeadSourceType.php

```diff
@@ -27,14 +27,18 @@ class CampaignLeadSourceType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $sourceType = $options['data']['sourceType'];
+        $sourceChoices = $options['source_choices'] ?? null;
```

Comment:
The default value should be an array as we iterate on it later on.
```suggestion
        $sourceChoices = $options['source_choices'] ?? [];
```